bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/AuthenticationApi.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.netatmo.internal.api;
+
+import static org.openhab.core.auth.oauth2client.internal.Keyword.*;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.jetty.http.HttpMethod;
+import org.openhab.binding.netatmo.internal.api.data.NetatmoConstants.FeatureArea;
+import org.openhab.binding.netatmo.internal.api.dto.NAAccessTokenResponse;
+import org.openhab.binding.netatmo.internal.config.NetatmoBindingConfiguration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Allows access to the AutomowerConnectApi
```

Comment:
automower ?

Reply:
Hmm, maybe related to:
- https://developer.husqvarnagroup.cloud/apis/Automower+Connect+API
- https://netatmostatic.blob.core.windows.net/static/press/releases/PM_Netatmo_and_Gardena_EN.pdf

?

Reply:
That would be a copy/paste of the class from another binding ?

Reply:
> That would be a copy/paste of the class from another binding ?

It does look like same text as in bundles/org.openhab.binding.automower/src/main/java/org/openhab/binding/automower/internal/rest/api/automowerconnect/AutomowerConnectApi.java, so seems like a mistake. Other than that I don't see any similarity. Strange. Maybe @clinique can enlighten us. :)

Reply:
Yes, I based my refactoring on some parts coming from Automower binding, everything has since changed but some comments were left forgotten :)

Reply:
Corrected